core/crypto/vrf/vrf.go

```diff
@@ -41,12 +41,10 @@ type PublicKey interface {
 	ProofToHash(m, proof []byte) (index [32]byte, err error)
 }
 
-// UniqueID computes a unique string for a directory, userID and appID combo.
-func UniqueID(userID, appID string) []byte {
+// UniqueID computes a unique string for a userID.
+func UniqueID(userID string) []byte {
 	b := new(bytes.Buffer)
```

Comment:
We can delete `UniqueID` now.  It's only purpose was to create a canonical string for the VRF function. 
VRF callers can pass in `userID` directly in place of `UniqueID`